### ISSUE TITLE
the trt can be used in c++?

### ISSUE THREAD

**Issue by henbucuoshanghai** [THUDM/FastLDM#1]
where is the cpp code?

**Comment by 1049451037** [comment]
No. The repo is based on Python.

**Comment by henbucuoshanghai** [comment]
get the trt engine,I can write the c++ version myself?

**Comment by 1049451037** [comment]
Yes, I think you can. The generated engine can be loaded by python or any other languages only if TensorRT provides the APIs.
